xau_main.py

```
@@ -0,0 +1,55 @@
+            print("\n--- Macro Analyst Report ---")
+            print(final_state["xau_macro_report"])
+
+        if final_state.get("xau_positioning_report"):
+            print("\n--- Positioning Analyst Report ---")
+            print(final_state["xau_positioning_report"])
+
+        print("\n--- Final Trade Decision ---")
+        print(decision)
+        print("="*80)
+
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)